WordPress/Classes/Services/RecentSitesService.swift

```diff
@@ -10,6 +10,7 @@ class RecentSitesService: NSObject {
     private let database: KeyValueDatabase
     private let databaseKey = "RecentSites"
     private let legacyLastUsedBlogKey = "LastUsedBlogURLDefaultsKey"
+    @objc static let WPTouchedBlogNotification = "WPTouchedBlogNotification"
```

Comment:
Since this is already namespaced by RecentSitesService, I'd drop the WP prefix. I was going to suggest replacing Blog with Site, but maybe just name it "Changed":

```
static let changedNotification = NSNotification.Name("RecentSitesChanged")
```